src/GuppyDataExplorer/GuppyDataExplorer.jsx

```diff
@@ -38,10 +39,32 @@ class GuppyDataExplorer extends React.Component {
       }
     }
 
+    if (this.props.filterConfig.tabs.length > 0) {
+      const defaultFilterArr = this.props.filterConfig.tabs.reduce((accumulator, currentValue) => {
+        const df = currentValue.defaultFilters;
+        if (df && df.length > 0) {
+          return [...accumulator, ...df];
+        }
+        return accumulator;
+      }, []);
+
+      defaultFilters = defaultFilterArr.reduce((tabAccumulator, tabDefaultFilter) => {
+        if (tabDefaultFilter.field && tabDefaultFilter.values.length > 0) {
+          // if it already exists add values to it
+          if (tabAccumulator[tabDefaultFilter.field]) {
+            tabAccumulator[tabDefaultFilter.field].selectedValues.push(...tabDefaultFilter.values);
+            return tabAccumulator;
+          }
+          return { ...tabAccumulator, [tabDefaultFilter.field]: { selectedValues: tabDefaultFilter.values } };
```

Comment:
I think these logic makes sense to me, but I also think this implies that `tabDefaultFilter.values` is required for each objects in `defaultFilters` and they have to be arrays
But I don't think we should do the validation here, maybe we should update the portal config validation script in our GH actions to catch this?

Reply:
the vales are required for setting the defaults, and must be in array so that the user can select multiple default values. This part of the code is added to handle the edge case of setting defaults for the same field in multiple tabs

Ill look into validating these new fields in the portal config validation script

Reply:
https://github.com/uc-cdis/gen3utils/blob/master/gen3utils/gitops/gitops_validator.py